dev/io.openliberty.microprofile.openapi.internal.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -20,7 +20,7 @@ mpOpenAPI=MicroProfile OpenAPI
 mpOpenAPI.desc=Configuration for MicroProfile OpenAPI.
 
 docPath=MicroProfile OpenAPI document endpoint path
-docPath.desc=Path to obtain OpenAPI 3.0/Swagger documents.
+docPath.desc=Specifies the path for obtaining OpenAPI documents. If no value is specified, the path is set to /openapi.
```

Comment:
If the default value is set in the metatype then we don't need to repeat it here in the description.

Reply:
The metatype defines a default for the docPath only. the UI Path does not given its default is relative to the docPath. so should remove the second part from docPath, but should be kept by uiPath